mteb/evaluation/evaluators/RetrievalEvaluator.py

```diff
@@ -77,10 +78,15 @@ def __init__(
         if "convert_to_tensor" not in encode_kwargs:
             encode_kwargs["convert_to_tensor"] = True
 
-        self.score_functions = {"cos_sim": cos_sim, "dot": dot_score}
+        self.score_functions = {
+            "cos_sim": cos_sim,
+            "dot": dot_score,
+            "max_sim": max_sim,
+        }
         self.score_function_desc = {
             "cos_sim": "Cosine Similarity",
             "dot": "Dot Product",
+            "max_sim": "MaxSim",
```

Comment:
Let's follow the rest and spell it out fully:
```suggestion
            "max_sim": "Max Similarity",
```